internal/pkg/manifest/transform.go

```diff
@@ -98,9 +99,36 @@ func (t buildArgsOrStringTransformer) Transformer(typ reflect.Type) func(dst, sr
 	}
 }
 
+type aliasTransformer struct{}
+
+// Transformer returns custom merge logic for Alias's fields.
+func (t aliasTransformer) Transformer(typ reflect.Type) func(dst, src reflect.Value) error {
+	if !typ.ConvertibleTo(reflect.TypeOf(Alias{})) {
```

Comment:
nit: this could just be
```suggestion
	if typ != reflect.TypeOf(Alias{}) {
		return nil
	}
```
since we don't have any type that's aliased to `Alias` type. (If we have any in the future, we are not 100% now sure whether we'd want to apply this transformer).